lib/sequencescape_client_v2.rb

```diff
@@ -1,20 +1,45 @@
 require "faraday"
 
 module SequencescapeClientV2
-
   class SequencescapeClientV2::Model < JsonApiClient::Resource
+    # Indicates if the asset should be synced back with the remove
+    class_attribute :sync
     # set the api base url in an abstract base class
     self.site = "#{Rails.configuration.ss_api_v2_uri}/api/v2/"
+    self.sync = false
   end
 
   class SequencescapeClientV2::Asset < SequencescapeClientV2::Model
   end
 
+  class SequencescapeClientV2::Labware < SequencescapeClientV2::Model
+    # The plural of labware is labware
+    def self.table_name
+      'labware'
+    end
+
+    has_many :receptacles
+
+    def sync?
+      %w[plates tube_racks].include? type
+    end
+
+    def wells
+      type == 'plates' ? receptacles : []
+    end
+
+    def racked_tubes
```

Comment:
The inelegant tube racks bit. Two bits to understand:
1) The client gem returns instances of labware, not tube racks, when using the joint endpoint, even though it has sufficient information to work out what to generate. 
2) Even ignoring this, we actually want to eager load each tube's contents, so would need to make another call anyway.

I don't think there is a cleaner solution here really. We *could* add racked tubes to the generic labware endpoint, but then we have to add the relationship to all other labware, where it doesn't apply.